File: src/targets/java/asynchttp/fixtures/postdata-malformed.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("POST", "https://httpbin.org/anything")
  .setHeader("content-type", "application/json")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();

File: src/targets/php/guzzle/fixtures/postdata-malformed.php
<?php
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'headers' => [
    'content-type' => 'application/json',
  ],
]);

echo $response->getBody();